src/test/java/uk/gov/laa/gpfd/config/builders/AuthorizeHttpRequestsBuilderTest.java

```
@@ -34,7 +34,7 @@ void shouldSwaggerYamlAccessibleWithoutAuthentication() throws Exception {
     @Test
     void shouldOpenAPIDocsDoesNotExist() throws Exception {
         mockMvc.perform(get("/v3/api-docs"))
-                .andExpect(status().isNotFound());
+                .andExpect(status().isOk());
```

Can we test the redirect url or something, to indicate we are showing the error in the ui and not just ignoring the error?

---

with refactored changes this change not required. reverted.